tests/api_test.py

```diff
@@ -69,6 +69,8 @@ def test_me(flask_client, dummy_uid):
         'email': 'example@example.com',
         'name': 'Guinea Pig',
         'uid': 'user123',
+        'auth_uid': 'user123',
+        'signature': 'C5jy3r5ykNuVrBioWLSXYzjxIAo',
```

Comment:
I'd use `mock_sign_user` instead like in the other tests involving APIs that return signed content